tests/test_recipient_validation.py

```diff
@@ -374,34 +359,16 @@ def test_validate_address_allows_any_non_empty_value_for_non_postcode(column):
     'address_line_4',
     'address_line_5',
     'address_line_6',
+    'postcode',
 ])
-def test_non_ascii_address_line_is_fine_for_non_postcode(column):
+def test_non_ascii_address_line_is_fine_for_all_columns(column):
     valid_address = u'\u041F\u0435\u0442\u044F'
-    assert validate_recipient(
-        valid_address, 'letter', column=column
-    ) == valid_address
-
-
-def test_validate_address_checks_postcode_is_real_uk_postcode(mocker):
-    is_a_real_uk_postcode_mock = mocker.patch('notifications_utils.recipients.is_a_real_uk_postcode')
-    is_a_real_uk_postcode_mock.return_value = True
-
-    assert validate_recipient('sw1 1aa', 'letter', column="postcode") == 'sw1 1aa'
-    assert is_a_real_uk_postcode_mock.called
-
-
-def test_validate_address_raises_if_postcode_is_not_real_uk_postcode(mocker):
-    is_a_real_uk_postcode_mock = mocker.patch('notifications_utils.recipients.is_a_real_uk_postcode')
-    is_a_real_uk_postcode_mock.return_value = False
-    with pytest.raises(InvalidAddressError) as e:
-        validate_recipient('90899', 'letter', column="postcode")
-    assert is_a_real_uk_postcode_mock.called
-    assert 'Not a real UK postcode' == str(e.value)
+    assert validate_recipient(valid_address, 'letter') == valid_address
```

Comment:
We aren't using `column` here either?

Reply:
I think this test has been removed in a subsequent commit.